lib/util/spmconfig.js

```diff
@@ -46,14 +61,26 @@ var envFileName = '/etc/sematext/receivers.config'
 **/
 loadEnvFromFile(envFileName)
 
+if (process.env.REGION !== undefined) {
+  setReceiverUrlsForRegion(process.env.REGION)
+}
+
 var spmDefaultConfig = {
   tokens: {
     spm: process.env.SPM_TOKEN || process.env.MONITORING_TOKEN,
     // prepare move to MONITORING_TOKEN and config.token.monitoring
     monitoring: process.env.MONITORING_TOKEN || process.env.SPM_TOKEN
   },
   influx: {
-    dbName: 'metrics'
+    dbName: 'metrics',
+    // example: "USER,PWD" adds user and working directory from ENV as tags
+    // example with static tags and ENV tags: "USER, customer_id=1234"
```

Comment:
$USER?  I know this is just a comment, but...

Reply:
No